io.catenax.planned_production_output/1.0.0/PlannedProductionOutput.ttl

```diff
@@ -0,0 +1,186 @@
+    samm:characteristic :DateTimeStamp;
+    samm:exampleValue "2023-04-01T14:23:00+01:00"^^xsd:date.
+:OrderIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Order ID Characteristic"@en;
+    samm:description "This is a proprietary order ID, generated by either the supplier side or customer side."@en;
+    samm:dataType xsd:string.
+:OrderPositionIdCharacteristic a samm:Characteristic;
+    samm:preferredName "Position ID within Order"@en;
+    samm:description "The position ID is unique for an order and used within business context to refer to a position."@en;
+    samm:dataType xsd:string.
+:QuantityCharacteristic a samm-c:Quantifiable;
+    samm:preferredName "Quantity Characteristic"@en;
+    samm:description "Describes the quantity and the unit of measure with which the planned and allocated production output is measured."@en;
+    samm:dataType :Quantity.
+:Quantity a samm:Entity;
+    samm:preferredName "Quantity"@en;
+    samm:description "Comprises the number of objects and the unit of measurement."@en;
+    samm:properties (:quantityNumber :unitOfMeasure).
+:unitOfMeasure a samm:Property;
+    samm:preferredName "Unit of Measure"@en;
+    samm:description "Unit of measurement for demand quantities."@en;
+    samm:characteristic :UnitOfMeasure;
+    samm:exampleValue "KGM".
+:UnitOfMeasure a samm-c:Enumeration;
+    samm:preferredName "Unit of Measure"@en;
+    samm:description "The Enumeration lists a selection of allowed units of measure referenced by their official code according to Codes for Units of Measure Used in International Trade"@en;
+    samm:dataType xsd:string;
+    samm:see <https://tfig.unece.org/contents/recommendation-20.htm>;
+    samm-c:values ("GRM" "KGM" "TNE" "STN" "ONZ" "LBR" "CMT" "MTR" "KTM" "INH" "FOT" "YRD" "CMK" "MTK" "INK" "FTK" "YDK" "CMQ" "MTQ" "INQ" "FTQ" "YDQ" "MLT" "LTR" "HLT" "H87" "SET" "PR" "ZP" "KWH" "").
+:quantityNumber a samm:Property;
+    samm:preferredName "Quantity Number"@en;
+    samm:description "The number of objects related to the unit of measure."@en;
+    samm:characteristic :NumberofObjects;
+    samm:exampleValue "2.5"^^xsd:double.
+:NumberofObjects a samm:Characteristic;
+    samm:preferredName "Number of Objects"@en;
+    samm:description "Quantifiable number of objects in reference to the unit of measure."@en;
+    samm:dataType xsd:double.
+:BPNSTrait a samm-c:Trait;
+    samm:preferredName "BPNS Trait"@en;
+    samm:description "Trait to ensure data format for BPNS."@en;
+    samm-c:baseCharacteristic :BPNS;
+    samm-c:constraint :BPNSRegularExpression.
+:BPNS a samm:Characteristic;
+    samm:preferredName "Business Partner Number Site"@en;
+    samm:description "BPNS represents a site which can be a production plant inside one location."@en;
+    samm:dataType xsd:string.
+:BPNSRegularExpression a samm-c:RegularExpressionConstraint;
```

Comment:
same here as with the uuid. We could think about extracting that to a shared aspect as it is already defined in a lot of different models